azure-pipelines.yml

```diff
@@ -19,11 +20,33 @@ jobs:
       vmImage: 'ubuntu-16.04'
     steps:
       - script: |
-          sudo apt-get install pkg-config libfuse-dev cmake libcurl4-gnutls-dev libgnutls28-dev uuid-dev libgcrypt20-dev libboost-all-dev gcc g++ -y
-          . ./build.sh
-          cp ./blobfuse $(Build.ArtifactStagingDirectory)
-        displayName: 'Generate blobfuse build'
+            sudo apt-get install pkg-config libfuse-dev cmake libcurl4-gnutls-dev libgnutls28-dev uuid-dev libgcrypt20-dev libboost-all-dev gcc g++ -y
+            . ./build.sh
+            cp ./blobfuse $(Build.ArtifactStagingDirectory)
+            cp ./blobfusetests $(Build.BinariesDirectory)
+        env:
+          INCLUDE_TESTS: 1
+        displayName: Generate blobfuse build for Ubuntu 16.04
+      - script: |
+            chmod 777 ./blobfusetests
+            . ./blobfusetests --gtest_filter=-BlobClientWrapperTest.*
+        workingDirectory: $(Build.BinariesDirectory)
+        condition: succeededOrFailed()        
+      - task: PublishBuildArtifacts@1
+        displayName: 'Publish Artifacts for blobfuse Ubuntu 16.04'
+        condition: succeededOrFailed()
+
+
+  - job: Linux_Build_1804   
+    pool:
+      vmImage:  'ubuntu-18.04'
+    steps:
+      - script: |
+            sudo apt-get install pkg-config libfuse-dev cmake libcurl4-gnutls-dev libgnutls28-dev uuid-dev libgcrypt20-dev libboost-all-dev gcc g++ -y
+            . ./build.sh
+            cp ./blobfuse $(Build.ArtifactStagingDirectory)
+        displayName: Generate blobfuse build for Ubuntu 18.04
       - task: PublishBuildArtifacts@1
-        displayName: 'Publish Artifacts'
```

Comment:
Do we have different set of steps of Ubuntu 16 and 18. If not there is no need to write two different jobs we can write a single one with strategy matrix and it will build both in parallel.

Reply:
It is all in this piece of code you are reading unless, I don't understand the question. Yes, a couple of steps are different.